server/main.lua

```diff
@@ -119,6 +119,24 @@ exports.ox_inventory:RegisterShop('Canteen', {
     inventory = Config.CanteenItems,
 })
 
+lib.addCommand('jailalarm', {
+    help = locale('info.jail_alarm'),
+    params = {
+        {
+            name = 'type',
+            type = 'string',
+            help = locale('info.jail_alarm')
+        }
+    },
+}, function(source, args)
+    if not exports.qbx_core:GetDutyCountType('leo') then return end
```

Comment:
GetDutyCountType does not return a boolean or nil at all, it returns a number representing the amount of people on duty for that type